### ISSUE TITLE
Filter out invalid xml characters

### ISSUE THREAD

**Issue by rotemkolus** [EPPlusSoftware/EPPlus#340]
By entering an invalid xml char to a cell- EPPlus does not filter it out, causing to the creation of a file which later cannot be open correctly.

**Example:**

```
public void EPPLUS_Filter_Out()
{
    var str = "a net￾work meta-analysis";
    var path = @"C:\EPPlus_Filter_Out_Invalid_Xml_Chars.xlsx";
    var stream = new MemoryStream();
    using (var excelPackage = new ExcelPackage())
    {
        var sheet = excelPackage.Workbook.Worksheets.Add("Dataset");
        sheet.SetValue(1, 2, str);
        stream.Position = 0;
        excelPackage.SaveAs(new FileInfo(path));
     }
}
```

**The created file:** (After pressing "Yes" opens as empty, if more valid records in this file- they're gone too)
![image](https://user-images.githubusercontent.com/32978761/112000031-40f8d700-8b26-11eb-96b6-2599c05dbe8b.png)

**Error:**
![image](https://user-images.githubusercontent.com/32978761/112000204-6e458500-8b26-11eb-88d5-9b1f72b43926.png)

**Comment by JanKallman** [comment]
Works fine when I try you sample. Maybe the character dissappeard when you pasted it into the issue? 
What is the invalid char you are using?

**Comment by rotemkolus** [comment]
This is the illegal char along with it's hexadecimal code
![image](https://user-images.githubusercontent.com/32978761/112426958-80e7d600-8d41-11eb-959c-7269827e45aa.png)

**Comment by JanKallman** [comment]
Fixed in version 5.6.2
